sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestConsumer.java

```diff
@@ -35,27 +40,57 @@ public class TestConsumer {
     @RegisterExtension
     public JdbiExtension h2Extension = JdbiExtension.h2().withInitializer(TestingInitializers.something()).withPlugin(new SqlObjectPlugin());
 
-    @Test
-    public void testReturnStream() {
-        Something one = new Something(3, "foo");
-        Something two = new Something(4, "bar");
-        Something thr = new Something(5, "baz");
+    Spiffy dao;
```

Comment:
private ?

Reply:
I do not usually bother to make test members private. It's an extra keyword that doesn't bring any benefit in this case.